psake.ps1

```diff
@@ -102,7 +102,7 @@ Task Build {
     }
 }
 
-Task Test -depends Build  {
+Task Test -depends Build {
```

Comment:
Running the "build" on Linux without Mono available meant that `dotnet461` wasn't available, so the build would fail, even though it isn't needed to run the tests when targeting `netcoreapp2.0`.  That was why building/testing in one step was the best way to success in all cases, because if `netcoreapp2.0` was available, then it would be able to build the dependencies.

I realize you modified the Travis CI scripts to include Mono, but it shouldn't be strictly necessary.